[2024-01-30 20:53:13,210] torch.distributed.run: [WARNING] 
[2024-01-30 20:53:13,210] torch.distributed.run: [WARNING] *****************************************
[2024-01-30 20:53:13,210] torch.distributed.run: [WARNING] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
[2024-01-30 20:53:13,210] torch.distributed.run: [WARNING] *****************************************
  File "/sailhome/jphilipp/research_projects/scai-tuning/experiments/hh_rlhf/train/train_sft.py", line 43
  File "/sailhome/jphilipp/research_projects/scai-tuning/experiments/hh_rlhf/train/train_sft.py", line 43
        get model
get model
                ^^^^^^^^^^

SyntaxErrorSyntaxError: : invalid syntaxinvalid syntax

  File "/sailhome/jphilipp/research_projects/scai-tuning/experiments/hh_rlhf/train/train_sft.py", line 43
    get model
        ^^^^^
SyntaxError: invalid syntax
  File "/sailhome/jphilipp/research_projects/scai-tuning/experiments/hh_rlhf/train/train_sft.py", line 43
    get model
        ^^^^^
SyntaxError: invalid syntax
[2024-01-30 20:53:18,222] torch.distributed.elastic.multiprocessing.api: [ERROR] failed (exitcode: 1) local_rank: 0 (pid: 472715) of binary: /scr/jphilipp/miniconda3/envs/scai-tuning/bin/python
Traceback (most recent call last):
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/bin/accelerate", line 8, in <module>
    sys.exit(main())
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/lib/python3.10/site-packages/accelerate/commands/accelerate_cli.py", line 47, in main
    args.func(args)
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/lib/python3.10/site-packages/accelerate/commands/launch.py", line 1008, in launch_command
    deepspeed_launcher(args)
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/lib/python3.10/site-packages/accelerate/commands/launch.py", line 724, in deepspeed_launcher
    distrib_run.run(args)
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/lib/python3.10/site-packages/torch/distributed/run.py", line 797, in run
    elastic_launch(
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 134, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/scr/jphilipp/miniconda3/envs/scai-tuning/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
train_sft.py FAILED
------------------------------------------------------------
Failures:
[1]:
  time      : 2024-01-30_20:53:18
  host      : cocoflops-hgx-1.stanford.edu
  rank      : 1 (local_rank: 1)
  exitcode  : 1 (pid: 472716)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
[2]:
  time      : 2024-01-30_20:53:18
  host      : cocoflops-hgx-1.stanford.edu
  rank      : 2 (local_rank: 2)
  exitcode  : 1 (pid: 472717)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
[3]:
  time      : 2024-01-30_20:53:18
  host      : cocoflops-hgx-1.stanford.edu
  rank      : 3 (local_rank: 3)
  exitcode  : 1 (pid: 472718)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2024-01-30_20:53:18
  host      : cocoflops-hgx-1.stanford.edu
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 472715)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
